index.html

```diff
@@ -0,0 +1,109 @@
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./icon/Enabled.png" alt="twenty icon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./icon/Disabled2.png" alt="Twitter icon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./icon/Disabled3.png" alt="M icon">
+                            </a>
+                        </li>
+                    </ul>   
+                    
+                </div>
+
+            </section>
+            
+            <section id="recent-works">    
+                <header id="recent-works-header">
+                    <h1>My Recent Works</h1>
+                    <div class="thin-line" ></div>
+            </header>
+
+            <div>
+                <div class="Featured" >
+
+                    <div class="Img-placeholder" >
+                        <a href="#">
+                            <img src="./images/Img-Placeholder.png" alt="Img-Placeholder">
+                        </a>
+                    </div>
+                    <div class="Right-block">
+                        <h3>Multi-Post Stories</h3>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        <div>
+                            <span class="tags" >css</span>
+                            <span class="tags" >html</span>
+                            <span class="tags" >Bootstrap</span>
+                            <span class="tags" >Ruby</span>
+                        </div>
+                        <button type="button" class="btn" >See Project</button>
+                    </div>
+                </div>
+        
+                <ul id="Card-container" >
+                    <li class="project-card">
+                        <div class="Card-description" >
+                            <h1>Profesional Art Printing Data</h1>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+
+                            <div>
+                                <span>html</span>
+                                <span>Bootstrap</span>
+                                <span>Ruby</span>
+                            </div>
+                        </div>
+                        <button type="button"></button>
+                    </li>
+
+                </ul>
+            </div>
+        </section>
+
+    </main>
+
+
+</body>
```

Comment:
- [ ] Please make sure to implement the about me section the whole section is missing from your design 
![image](https://user-images.githubusercontent.com/74744787/164945923-6d30575c-64db-439b-a20f-ae3d71e007b8.png)